app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt

```diff
@@ -208,13 +215,30 @@ open class BaseEditTextFormField(context: Context, private val attrs: AttributeS
 
     override fun onFocusChange(v: View?, hasFocus: Boolean) {
         if (!hasFocus) {
-            val isValid = isValid()
+            handleNoFocusOnView()
+            hidePlaceholder()
+        } else {
+            showPlaceholder()
+        }
+    }
 
-            if (isValid.error.isNotEmpty()) {
-                showError(isValid.error)
-            } else {
-                clearError()
-            }
+    private fun showPlaceholder() {
+        editText?.postDelayed({
```

Comment:
Is this **editText?** the same as **textInputLayout?.editText?** ??

Reply:
sii